app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -251,6 +270,17 @@ private void navigateTo(Class<? extends GdgActivity> activityClass, Bundle addit
         mDrawerLayout.closeDrawers();
     }
 
+    protected void setNavDrawerItemSelected(String title) {
+        Menu menu = mNavigationView.getMenu();
```

Comment:
The problem I was having using the ID is that the SPECIAL_EVENT items have the same id therefore clicking on Study Jam would check the WTM, etc.. By using their title string makes it be able to find.